fastchat/serve/gradio_web_server.py

```diff
@@ -527,7 +556,7 @@ def bot_response(
             "finish": round(finish_tstamp, 4),
             "state": state.dict(),
             "ip": get_ip(request),
-            "images": images_hash,
+            "images": image_filenames,
```

Comment:
Should it be put into the conversation logic as well?

Reply:
Nice catch! I think this was written under the original assumption that the image hashes are not stored in the `state.dict()` which is a new feature added in #3010. I think a good idea may be to first merge this PR then in #3010, I can fix the merge conflicts by placing the image filenames in the `state.dict()` similar to the code [here](https://github.com/lm-sys/FastChat/blob/5e11dbf762f9239414f410295c4fcedae326bd22/fastchat/conversation.py#L504).

Reply:
Sounds good. I'll merge this one.